diff --git a/.gitignore b/.gitignore
old mode 100644
new mode 100755
diff --git a/composer.json b/composer.json
old mode 100644
new mode 100755
diff --git a/config/install/stripebyhabeuk.stripepaiement.yml b/config/install/stripebyhabeuk.stripepaiement.yml
old mode 100644
new mode 100755
diff --git a/config/schema/stripebyhabeuk.schema.yml b/config/schema/stripebyhabeuk.schema.yml
old mode 100644
new mode 100755
diff --git a/js/Copy-stripejsinit.js b/js/Copy-stripejsinit.js
old mode 100644
new mode 100755
diff --git a/js/stripejsinit.js b/js/stripejsinit.js
old mode 100644
new mode 100755
diff --git a/reliquat_to_paid.page.inc b/reliquat_to_paid.page.inc
old mode 100644
new mode 100755
diff --git a/src/Controller/ReliquatToPaidController.php b/src/Controller/ReliquatToPaidController.php
old mode 100644
new mode 100755
diff --git a/src/Controller/StripebyhabeukController.php b/src/Controller/StripebyhabeukController.php
old mode 100644
new mode 100755
index 0899dee..f6a5e5e
--- a/src/Controller/StripebyhabeukController.php
+++ b/src/Controller/StripebyhabeukController.php
@@ -18,18 +18,18 @@ use Symfony\Component\HttpFoundation\Request;
  * Returns responses for stripe by habeuk routes.
  */
 class StripebyhabeukController extends ControllerBase {
-
+  
   /**
    * The minor units converter.
    *
    * @var \Drupal\commerce_price\MinorUnitsConverterInterface
    */
   protected $minorUnitsConverter;
-
+  
   function __construct(MinorUnitsConverterInterface $minor_units_converter) {
     $this->minorUnitsConverter = $minor_units_converter;
   }
-
+  
   /**
    *
    * {@inheritdoc}
@@ -37,13 +37,13 @@ class StripebyhabeukController extends ControllerBase {
   public static function create(ContainerInterface $container) {
     return new static($container->get('commerce_price.minor_units_converter'));
   }
-
+  
   /**
    *
    * @var string
    */
   protected $endpointSecret = '';
-
+  
   /**
    * Permet d'analyser les methodes de paiement.
    */
@@ -52,7 +52,6 @@ class StripebyhabeukController extends ControllerBase {
     $query = $this->entityTypeManager()->getStorage('commerce_payment_method')->getQuery();
     $query->pager(20);
     $query->sort('created', 'DESC');
-    $query->accessCheck(False);
     $ids = $query->execute();
     $header = [
       'id' => '#id',
@@ -106,7 +105,7 @@ class StripebyhabeukController extends ControllerBase {
     ];
     return $build;
   }
-
+  
   /**
    * Permet d'effectuer les paiements reliquat_to_paid.
    * ( Mais il faudra que cela se fasse au niveau de l'entiter, ainsi on pourra
@@ -135,13 +134,14 @@ class StripebyhabeukController extends ControllerBase {
         ]
       ];
       $paymentIntent = $this->createPaymentIntent($ReliquatToPaid);
-    } else {
+    }
+    else {
       $this->messenger()->addStatus(" Aucun relicat pour cette commande ");
     }
-
+    
     return $build;
   }
-
+  
   /**
    * Permet de creer ou de mettre à jour PaymentIntent.
    * S'assure egalement que pour une commande on a un unique paymentIntent.
@@ -177,7 +177,7 @@ class StripebyhabeukController extends ControllerBase {
      * @var \Drupal\stripebyhabeuk\Plugin\Commerce\PaymentGateway\StripeAcompte $stripebyhabeuk
      */
     $stripebyhabeuk = $PaymentGatewayManager->createInstance('stripebyhabeuk_acompte');
-
+    
     $order = \Drupal\commerce_order\Entity\Order::load($entity->get('commerce_order')->target_id);
     if ($order) {
       // $customer_remote_id =
@@ -186,7 +186,7 @@ class StripebyhabeukController extends ControllerBase {
       // $intent_array['customer'] = $customer_remote_id;
       // }
     }
-
+    
     $GateWay = new GateWay($stripebyhabeuk->getSecretKey());
     /**
      *
@@ -201,7 +201,8 @@ class StripebyhabeukController extends ControllerBase {
       // cette sauvegarde de order, contient des informations prevenant de
       // amount().
       $entity->set('payment_intent_id', $paymentIntents->id);
-    } else {
+    }
+    else {
       /**
        * On verifie si l'utilisateur a deja payé ou pas.
        */
@@ -215,7 +216,7 @@ class StripebyhabeukController extends ControllerBase {
     $entity->save();
     return $paymentIntents;
   }
-
+  
   /**
    * --
    */
@@ -226,7 +227,7 @@ class StripebyhabeukController extends ControllerBase {
     ];
     return $this->reponse($configs, 200);
   }
-
+  
   /**
    *
    * @param array|string $configs
@@ -243,7 +244,7 @@ class StripebyhabeukController extends ControllerBase {
     $reponse->setContent($configs);
     return $reponse;
   }
-
+  
   /**
    * Builds a renderable list of operation links for the entity.
    *
@@ -259,10 +260,10 @@ class StripebyhabeukController extends ControllerBase {
       '#type' => 'operations',
       '#links' => $this->getOperations($request, $entity)
     ];
-
+    
     return $build;
   }
-
+  
   /**
    *
    * {@inheritdoc}
@@ -285,7 +286,7 @@ class StripebyhabeukController extends ControllerBase {
     }
     return $operations;
   }
-
+  
   /**
    * Ensures that a destination is present on the given URL.
    *
@@ -301,4 +302,7 @@ class StripebyhabeukController extends ControllerBase {
       ]
     ]);
   }
+  
 }
+  
+
diff --git a/src/Entity/ReliquatToPaid.php b/src/Entity/ReliquatToPaid.php
old mode 100644
new mode 100755
diff --git a/src/Entity/ReliquatToPaidInterface.php b/src/Entity/ReliquatToPaidInterface.php
old mode 100644
new mode 100755
diff --git a/src/Entity/ReliquatToPaidViewsData.php b/src/Entity/ReliquatToPaidViewsData.php
old mode 100644
new mode 100755
diff --git a/src/ErrorHelper.php b/src/ErrorHelper.php
old mode 100644
new mode 100755
diff --git a/src/EventSubscriber/ListPaiement.php b/src/EventSubscriber/ListPaiement.php
old mode 100644
new mode 100755
diff --git a/src/Form/ReliquatToPaidDeleteForm.php b/src/Form/ReliquatToPaidDeleteForm.php
old mode 100644
new mode 100755
diff --git a/src/Form/ReliquatToPaidForm.php b/src/Form/ReliquatToPaidForm.php
old mode 100644
new mode 100755
diff --git a/src/Form/ReliquatToPaidRevisionDeleteForm.php b/src/Form/ReliquatToPaidRevisionDeleteForm.php
old mode 100644
new mode 100755
diff --git a/src/Form/ReliquatToPaidRevisionRevertForm.php b/src/Form/ReliquatToPaidRevisionRevertForm.php
old mode 100644
new mode 100755
diff --git a/src/Form/ReliquatToPaidRevisionRevertTranslationForm.php b/src/Form/ReliquatToPaidRevisionRevertTranslationForm.php
old mode 100644
new mode 100755
diff --git a/src/Form/ReliquatToPaidSettingsForm.php b/src/Form/ReliquatToPaidSettingsForm.php
old mode 100644
new mode 100755
diff --git a/src/Form/SettingsForm.php b/src/Form/SettingsForm.php
old mode 100644
new mode 100755
diff --git a/src/Form/StripePaiementForm.php b/src/Form/StripePaiementForm.php
old mode 100644
new mode 100755
diff --git a/src/Handler/StripebyHabeukPaymentInformation.php b/src/Handler/StripebyHabeukPaymentInformation.php
old mode 100644
new mode 100755
diff --git a/src/Plugin/Block/ExampleBlock.php b/src/Plugin/Block/ExampleBlock.php
old mode 100644
new mode 100755
diff --git a/src/Plugin/Commerce/CheckoutPane/StripebyhabeukReview.php b/src/Plugin/Commerce/CheckoutPane/StripebyhabeukReview.php
old mode 100644
new mode 100755
diff --git a/src/Plugin/Commerce/PaymentGateway/StripeAcompte.php b/src/Plugin/Commerce/PaymentGateway/StripeAcompte.php
old mode 100644
new mode 100755
diff --git a/src/Plugin/Commerce/PaymentGateway/StripebyHabeukAcompteInterface.php b/src/Plugin/Commerce/PaymentGateway/StripebyHabeukAcompteInterface.php
old mode 100644
new mode 100755
diff --git a/src/Plugin/Commerce/PaymentGateway/StripebyHabeukInterface.php b/src/Plugin/Commerce/PaymentGateway/StripebyHabeukInterface.php
old mode 100644
new mode 100755
diff --git a/src/Plugin/Commerce/PaymentGateway/Stripebyhabeuk.php b/src/Plugin/Commerce/PaymentGateway/Stripebyhabeuk.php
old mode 100644
new mode 100755
index f0baa5d..fd70bb1
--- a/src/Plugin/Commerce/PaymentGateway/Stripebyhabeuk.php
+++ b/src/Plugin/Commerce/PaymentGateway/Stripebyhabeuk.php
@@ -45,7 +45,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
    * @var string
    */
   protected $CurrencySymbol = NULL;
-
+  
   public function deletePaymentMethod(PaymentMethodInterface $payment_method) {
     // Delete the remote record.
     $payment_method_remote_id = $payment_method->getRemoteId();
@@ -60,12 +60,13 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
       if ($remote_payment_method->customer) {
         $remote_payment_method->detach();
       }
-    } catch (ApiErrorException $e) {
+    }
+    catch (ApiErrorException $e) {
       ErrorHelper::handleException($e);
     }
     $payment_method->delete();
   }
-
+  
   /**
    * Elle se charge d'effectuer le paiement via l'API et on cree l'entite
    * payement si le paiement a reussi ou est en attente.
@@ -99,7 +100,8 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
       if (is_object($paymentIntents->last_payment_error)) {
         $error = $paymentIntents->last_payment_error;
         $decline_message = sprintf('%s: %s', $error->type, $error->message ?? '');
-      } else {
+      }
+      else {
         $decline_message = $paymentIntents->last_payment_error;
       }
       throw new HardDeclineException($decline_message);
@@ -116,7 +118,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     // return $paymentIntents, help order module that extend this module.
     return $paymentIntents;
   }
-
+  
   /**
    * Permet de creer ou de mettre à jour PaymentIntent.
    * S'assure egalement que pour une commande on a un unique paymentIntent.
@@ -140,7 +142,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
       'payment_method' => $payment_method->getRemoteId(),
       'capture_method' => 'automatic'
     ];
-
+    
     $customer_remote_id = $this->getRemoteCustomerId($order->getCustomer());
     if (!empty($customer_remote_id)) {
       $intent_array['customer'] = $customer_remote_id;
@@ -159,7 +161,8 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
       // cette sauvegarde de order, contient des informations prevenant de
       // amount().
       $order->setData('stripebyhabeuk_payment_intent_id', $paymentIntents->id);
-    } else {
+    }
+    else {
       /**
        * On verifie si l'utilisateur a deja payé ou pas.
        */
@@ -176,7 +179,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     $order->save();
     return $paymentIntents;
   }
-
+  
   /**
    *
    * {@inheritdoc}
@@ -185,7 +188,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
   public function amount(Price $amount, OrderInterface $order) {
     return $this->minorUnitsConverter->toMinorUnits($amount);
   }
-
+  
   public function getCurrencyCode(OrderInterface $order) {
     if ($this->CurrencyCode === NULL) {
       $amount = $order->getTotalPrice();
@@ -193,7 +196,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     }
     return $this->CurrencyCode;
   }
-
+  
   public function getCurrencySymbol(OrderInterface $order) {
     if ($this->CurrencySymbol === NULL) {
       $code = $this->getCurrencyCode($order);
@@ -205,7 +208,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     }
     return $this->CurrencySymbol;
   }
-
+  
   /**
    * Cette methode est executé apres la validation qui presente les methodes
    * de paiements.
@@ -223,19 +226,20 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     ];
     foreach ($required_keys as $required_key) {
       if (empty($payment_details[$required_key])) {
-
+        
         throw new InvalidRequestException(sprintf('$payment_details must contain the %s key.', $required_key));
       }
     }
-
+    
     if (!empty($payment_details['stripebyhabeuk_save_cb'])) {
-      // \drupal::messenger()->addStatus('reutilisable : oui ');
+      \drupal::messenger()->addStatus('reutilisable : oui ');
       $payment_method->setReusable(TRUE);
-    } else {
-      // \drupal::messenger()->addStatus('reutilisable : non ');
+    }
+    else {
+      \drupal::messenger()->addStatus('reutilisable : non ');
       $payment_method->setReusable(false);
     }
-
+    
     $remote_payment_method = $this->UpdatePaymentMethods($payment_method, $payment_details);
     $payment_method->card_type = $this->mapCreditCardType($remote_payment_method['brand']);
     $payment_method->card_number = $remote_payment_method['last4'];
@@ -246,7 +250,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     $payment_method->setExpiresTime($expires);
     $payment_method->save();
   }
-
+  
   /**
    * Permet de completer les informations sur la methode de paiment (ajouter le
    * client, ajouter le billing information).
@@ -289,10 +293,11 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
         $customer_id = $customer->id;
         $this->setRemoteCustomerId($owner, $customer_id);
         $owner->save();
-      } else {
+      }
+      else {
         $email = NULL;
       }
-
+      
       if ($customer_id && $email) {
         $payment_method_data = [
           'email' => $email
@@ -314,11 +319,12 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
         ]);
       }
       return $stripe_payment_method->card;
-    } catch (ApiErrorException $e) {
+    }
+    catch (ApiErrorException $e) {
       ErrorHelper::handleException($e);
     }
   }
-
+  
   /**
    *
    * {@inheritdoc}
@@ -328,24 +334,24 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     $operations = parent::buildPaymentOperations();
     return $operations;
   }
-
+  
   public function getPublishableKey() {
     return $this->configuration['publishable_key'];
   }
-
+  
   public function getSecretKey() {
     return $this->configuration['secret_key'];
   }
-
+  
   public function voidPayment(PaymentInterface $payment) {
   }
-
+  
   public function capturePayment(PaymentInterface $payment, Price $amount = NULL) {
   }
-
+  
   public function refundPayment(PaymentInterface $payment, Price $amount = NULL) {
   }
-
+  
   /**
    *
    * {@inheritdoc}
@@ -358,7 +364,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
       'enable_credit_card_icons' => true
     ] + parent::defaultConfiguration();
   }
-
+  
   /**
    *
    * {@inheritdoc}
@@ -374,7 +380,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
       '#default_value' => $this->configuration['publishable_key'],
       '#required' => TRUE
     ];
-
+    
     $form['secret_key'] = [
       '#type' => 'textfield',
       '#title' => $this->t('Secrete key'),
@@ -382,16 +388,16 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
       '#default_value' => $this->configuration['secret_key'],
       '#required' => TRUE
     ];
-
+    
     $form['enable_credit_card_icons'] = [
       '#type' => 'checkbox',
       '#title' => $this->t('Enable credit cart icons'),
       '#default_value' => $this->configuration['enable_credit_card_icons']
     ];
-
+    
     return $form;
   }
-
+  
   /**
    *
    * {@inheritdoc}
@@ -404,14 +410,16 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
       try {
         $GateWayTest = new GateWay($values['secret_key'], $values['publishable_key']);
         $GateWayTest->testValidationKey($expected_livemode);
-      } catch (ExceptionStripe $e) {
+      }
+      catch (ExceptionStripe $e) {
         $form_state->setError($form['secret_key'], $e->getMessage());
-      } catch (ApiErrorException $e) {
+      }
+      catch (ApiErrorException $e) {
         $form_state->setError($form['secret_key'], $e->getMessage());
       }
     }
   }
-
+  
   /**
    *
    * {@inheritdoc}
@@ -424,7 +432,7 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     $this->configuration['publishable_key'] = $values['publishable_key'];
     $this->configuration['enable_credit_card_icons'] = $values['enable_credit_card_icons'];
   }
-
+  
   /**
    * Maps the Stripe credit card type to a Commerce credit card type.
    *
@@ -448,4 +456,5 @@ class Stripebyhabeuk extends OnsitePaymentGatewayBase implements StripebyHabeukI
     }
     return $map[$card_type];
   }
-}
+  
+}
\ No newline at end of file
diff --git a/src/PluginForm/StripebyhabeukCheckoutForm.php b/src/PluginForm/StripebyhabeukCheckoutForm.php
old mode 100644
new mode 100755
diff --git a/src/ReliquatToPaidAccessControlHandler.php b/src/ReliquatToPaidAccessControlHandler.php
old mode 100644
new mode 100755
diff --git a/src/ReliquatToPaidHtmlRouteProvider.php b/src/ReliquatToPaidHtmlRouteProvider.php
old mode 100644
new mode 100755
diff --git a/src/ReliquatToPaidListBuilder.php b/src/ReliquatToPaidListBuilder.php
old mode 100644
new mode 100755
diff --git a/src/ReliquatToPaidStorage.php b/src/ReliquatToPaidStorage.php
old mode 100644
new mode 100755
diff --git a/src/ReliquatToPaidStorageInterface.php b/src/ReliquatToPaidStorageInterface.php
old mode 100644
new mode 100755
diff --git a/src/ReliquatToPaidTranslationHandler.php b/src/ReliquatToPaidTranslationHandler.php
old mode 100644
new mode 100755
diff --git a/src/Services/PasserelleStripe.php b/src/Services/PasserelleStripe.php
old mode 100644
new mode 100755
diff --git a/src/Services/StripebyhabeukLateOrderProcessor.php b/src/Services/StripebyhabeukLateOrderProcessor.php
old mode 100644
new mode 100755
diff --git a/src/Services/webformStripe.php b/src/Services/webformStripe.php
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.commerce_adjustment_types.yml b/stripebyhabeuk.commerce_adjustment_types.yml
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.info.yml b/stripebyhabeuk.info.yml
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.libraries.yml b/stripebyhabeuk.libraries.yml
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.links.action.yml b/stripebyhabeuk.links.action.yml
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.links.menu.yml b/stripebyhabeuk.links.menu.yml
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.links.task.yml b/stripebyhabeuk.links.task.yml
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.module b/stripebyhabeuk.module
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.permissions.yml b/stripebyhabeuk.permissions.yml
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.routing.yml b/stripebyhabeuk.routing.yml
old mode 100644
new mode 100755
diff --git a/stripebyhabeuk.services.yml b/stripebyhabeuk.services.yml
old mode 100644
new mode 100755
diff --git a/templates/reliquat_to_paid.html.twig b/templates/reliquat_to_paid.html.twig
old mode 100644
new mode 100755
